### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by TIove**

### CODE REVIEW

File: HW3/Menues/Database/Books.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public void Delete(SqlConnection connection)
             string nameForDelete = Console.ReadLine();
 
             string query;
-            if (string.IsNullOrEmpty(nameForDelete))
+            if (string.IsNullOrEmpty(nameForDelete)) // can be shorter, using ?: ternary operator
             {
                 query = $"delete from Books";
```
Comment (vorobevamv) [inline HW3/Menues/Database/Books.cs]: Better write commands in upper-case (DELETE, INSERT etc)

File: HW3/Menues/Database/Books.cs [hunk 2]
```diff
@@ -94,11 +94,11 @@ public void Update(SqlConnection connection)
                 query = $"update Books set name = '{newName}' where name='{nameForUpdate}'";
             }
 
-            using (SqlCommand comm = new SqlCommand(query, connection))
+            using (SqlCommand command = new SqlCommand(query, connection))
```
Comment (vorobevamv) [inline HW3/Menues/Database/Books.cs:96]: to line 88:
there are three types of empty: null, "", "   ".
for catching all of them use IsNullOrWhiteSpace; IsNullOrEmpty don't catch "   "

File: HW3/Menues/Database/Books.cs [hunk 3]
```diff
@@ -21,18 +21,18 @@ public void Create(SqlConnection connection)
 
             ConsoleHelper.WriteService("Enter library of the book");
             string input = Console.ReadLine();
-            if(!int.TryParse(input, out int id) && id > 0)
-            {
+            if(!int.TryParse(input, out int id) && id > 0) // it seems like broken logic. If we can parse number, we won't check the id > 0. If we cannot parse number, then we check if int default > 0.
```
Comment (vorobevamv) [inline HW3/Menues/Database/Books.cs]: to lines 20, 23 and so on:
better separate logic between methods; so 'Create' should create record in DB, it shouldn't check users' inputs; I think, receiving data for query is better move to other method

File: HW3/Menues/Database/Books.cs [hunk 4]
```diff
@@ -112,18 +112,16 @@ public void Update(SqlConnection connection)
         public void Read(SqlConnection connection)
         {
             connection.Open();
-            string query = @"Select * from Books";
+            string query = @"select * from Books"; // You should write all commands low-case or UPPER-CASE
 
             using (SqlCommand comm = new SqlCommand(query, connection))
+            using (SqlDataReader read = comm.ExecuteReader()) // There was too many deep of brackets. Now it's prettier
```
Comment (vorobevamv) [inline HW3/Menues/Database/Books.cs]: mmm.. prettier - yes, but for me it's easier to read with brackets

File: HW3/Menues/Database/DataBaseMenu.cs [hunk 5]
```diff
@@ -47,18 +47,19 @@ protected override void SelectMenuItem()
             ShowCommands();
         }
 
-        public void ShowCommands()
+        private void ShowCommands()
```
Comment (vorobevamv) [inline HW3/Menues/Database/DataBaseMenu.cs:50]: lines 22-26
may be put it into 'default'?

File: HW3/Menues/Database/Library.cs [hunk 6]
```diff
@@ -20,17 +20,18 @@ public void Create(SqlConnection connection)
             string address = Console.ReadLine();
 
             string query = @$"insert into Libraries values ('{address}') ";
-            using (SqlCommand comm = new SqlCommand(query, connection))
+            using (SqlCommand command = new SqlCommand(query, connection))
```
Comment (vorobevamv) [inline HW3/Menues/Database/Library.cs:20]: line 22:
'@' - what for?

File: HW3/Menues/Database/DataBaseMenu.cs (hunk 5 above)
Comment (TheDioBrando) [reply HW3/Menues/Database/DataBaseMenu.cs:50]: What is 'default'?

File: HW3/Menues/Database/Books.cs (hunk 1 above)
Comment (TheDioBrando) [reply HW3/Menues/Database/Books.cs]: Done

File: HW3/Menues/Database/Books.cs (hunk 2 above)
Comment (TheDioBrando) [reply HW3/Menues/Database/Books.cs:96]: Done

File: HW3/Menues/Database/Books.cs (hunk 3 above)
Comment (TheDioBrando) [reply HW3/Menues/Database/Books.cs]: Done

File: HW3/Menues/Database/Library.cs (hunk 6 above)
Comment (TheDioBrando) [reply HW3/Menues/Database/Library.cs:20]: Removed @

File: HW3/Menues/Database/Books.cs (hunk 4 above)
Comment (TheDioBrando) [reply HW3/Menues/Database/Books.cs]: Left Igor's style

File: HW3/Menues/Database/DataBaseMenu.cs (hunk 5 above)
Comment (TheDioBrando) [reply HW3/Menues/Database/DataBaseMenu.cs:50]: i'd rather leave check before switch

**Pull request merged**